ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/Deduplicate.java

```diff
@@ -105,6 +109,23 @@ public static Result of(PCollectionTuple tuple, TupleTag<PubsubMessage> outputTa
             errorTag());
       }
 
+      /**
+       * Strip the payload from duplicate messages and add them to the error collection, returning
+       * a {@link Result} of the non-duplicate output and the error collection.
+       */
+      public WithErrors.Result<PCollection<PubsubMessage>> sendDuplicateMetadataToErrors() {
+        PCollection<PubsubMessage> duplicateMetadata = tuple().get(duplicateTag())
+            .apply("DropDuplicatePayloads", MapElements //
+                .into(TypeDescriptor.of(PubsubMessage.class))
+                .via(message -> FailureMessage.of("Duplicate",
```

Comment:
Passing the literal "Duplicate" to be populated as "exception_type". Per @trink, this description is more obvious than "Deduplicate.RemoveDuplicates"